test/lib/rules/validate-jsdoc/require-description-complete-sentence.js

```diff
@@ -105,8 +105,8 @@ describe('lib/rules/validate-jsdoc/require-description-complete-sentence', funct
                     }
                 },
                 errors: {
-                    line: 2,
-                    column: 3
+                    line: 1,
+                    column: 0
```

Comment:
Description offsets needs to be calculated and maintained. Didn't have too much time to implement it from scratch for DocTags.